torchtext/data/batch.py

```diff
@@ -1,7 +1,7 @@
 import torch
 
 
-class Batch(object):
+class Batch:
```

Comment:
Is this relevant to the drop of python2 support?

Reply:
Reverted.